docs/client.md

```diff
@@ -0,0 +1,29 @@
+# General Requirements
+
+Designing and delivering a hackweek can be a significant undertaking! Here are some initial considerations when preparing your event: 
+
+## Location 
+
+Hackweeks require spaces for open interaction and collaboration. Other considerations include  accessibility, providing resources for child care and having flexibile scheduling (especially for virtual work). We have supported a variety of venues in the past and can work with you to find the best option.
+
+### In Person
+
+We look for spaces that offer fast internet connections, opportunities for random interactions during informal sessions, and interactive exchanges during tutorials and projects.
+
+### Virtual 
+
+We use a variety of platforms including Zoom, wonder.me, Slack and JupyterHub to create accessible and interactive virtual spaces.
+
+### Hybrid of in-person and virtual
```

Comment:
minor styling comment: it's nice to be consistent with capitalization in headers, caps for each word? `Hybrid of in-person and virtual` or `Hybrid of In-person and Virtual`